### DISCUSSION TITLE
Add custom keybindings

### DISCUSSION THREAD

**Discussion by ArchGalileu** [abougouffa/minemacs#98] [Q&A] [answered]
Hello

Trying to add keybindings to +map-local 'org-mode-map on my config.el without sucess.

On me-org.el i can add but for organization porpuses and updates i would like to add to config.el.

Any advice?

Congratulations for this incredible project, for me is as best program i use, before was doom, but this is very clean, fast. and elegant .. ;)

Best regards
ArchGalileu

**Comment by abougouffa** [comment, marked as answer]
Hello @ArchGalileu

For custom keybindings, there are two types: global or local (mode-specific) keybindings.

For the global ones, you can simply add the `+map!` block to your `config.el`. If you want to overwrite the default bindings, it is preferred to wrap the `+map!` block in an `(with-eval-after-load 'me-general-ready (+map! "a" #'some-command))`.

For the local keybindings (set with `+map-local!`), you can do it after loading the concerned package. Something like:

```elisp
(with-eval-after-load 'org ;; After loading the org package
  ;; Set the local keybindings
  (+map-local! :keymaps 'org-mode-map
    "xx" #'some-command
    "xy" #'another-command))
```

Thanks for the compliment, happy to hear that you've found my config useful!

**Reply by ArchGalileu** [reply]
Thank you ;)

Perfect 

Best regards
ArchGalileu
